src/scanner.cc

```diff
@@ -1,18 +1,19 @@
 #include <tree_sitter/parser.h>
```

Comment:
Would it be helpful to add a comment describing how the file was formatted so that it will be easier to maintain formatting in the future and prevent larger formatting-only diffs?

Reply:
Good point. I chose Google-style because that led to the fewest re-formatted lines.